javascript/src/main/pages/Schedule/PersonalSchedule.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { useEffect } from "react";
+import { Jumbotron } from "react-bootstrap";
+import PersonalScheduleTable from "main/components/Schedule/PersonalScheduleTable";
+
+import { useAuth0 } from "@auth0/auth0-react";
+import useSWR from "swr";
+import { fetchWithToken } from "main/utils/fetch";
+import {
+  buildDeleteSchedule
+} from "main/services/Schedule/scheduleServices";
+import { useParams } from "react-router-dom";
+import { useToasts } from 'react-toast-notifications'
+
+const PersonalSchedule = () => {
+  const { addToast } = useToasts();
+  const { getAccessTokenSilently: getToken} = useAuth0();
+  const { scheduleId } = useParams();
+  const { data: schedule } = useSWR([`/api/member/schedule/get/${scheduleId}`, getToken], fetchWithToken);
+  const { mutate: mutateSchedules } = useSWR(["/api/member/schedule/getSchedules", getToken], fetchWithToken);
+  console.log("schedule=", schedule);
+
+  useEffect(() => {
+    mutateSchedules();
+  }, [mutateSchedules]);
+
+  const deleteSchedule = buildDeleteSchedule(
+    getToken,
+    (response) => {
+      if (response.error) {
+        console.log("error message: ", response.error);
+        addToast(response.error, { appearance: 'error' });
+      }
+      else {
+        mutateSchedules();
+        addToast("Schedule deleted", { appearance: 'success' });
+      }
+    },
+    (err) => {
+      console.log("error message: ", err);
+      addToast("Error deleting schedule", { appearance: 'error' });
+    }
+  );
+
+  var singleSchedule = [];
+  singleSchedule.push(schedule);

```

Comment:
This can just be written as 

```
var singleSchedule = [schedule]
```

Or better yet, since singleSchedule is only used when you pass in the props on line 50 below, just delete these two lines, and use `data={[schedule]}`